docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -14,13 +14,40 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.6.1>>
 * <<release-notes-8.6.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.6.1 relnotes
```

Comment:
The known issue for Osquery results being delayed needs to carry into 8.6.1, we didn't have enough time to fix it for this release: https://github.com/elastic/observability-docs/blob/9e5574bc06839b9d49a01e1b6f852f13d7bcaff4/docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc?plain=1#L69-L85